React_pakadmin/tar11_3/src/GuessNumber.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from "react";
+
+export default function GuessNumber({ randNumber }) {
+  const [totalGuesses, setTotalGuesses] = useState(0);
+  const [outputMessage, setOutputMessage] = useState("");
+
+  function onNumberChange(e) {
+    const guessingNumber = e.target.value;
+    setTotalGuesses(totalGuesses + 1);
+    if (guessingNumber > randNumber) {
+      setOutputMessage(
+        totalGuesses % 10 === 0 ? "bigger than" : "smaller than"
```

Comment:
I like this fake random elegant solution.

Reply:
🥇